pkgs/development/python-modules/numpy-groupies/default.nix

```
@@ -0,0 +1,44 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchFromGitHub,
+  pytestCheckHook,
+  setuptools,
+  setuptools-scm,
+  numpy,
+  numba,
+  pandas,
+}:
+
+buildPythonPackage rec {
+  pname = "numpy-groupies";
+  version = "0.11.2";
+  pyproject = true;
+
+  src = fetchFromGitHub {
+    owner = "ml31415";
+    repo = pname;
```

```suggestion
    repo = "numpy-groupies";
```
This is a misuse of `pname`. `repo` and `pname` are semantically different.